eng/pipelines/common/platform-matrix.yml

```diff
@@ -445,6 +445,30 @@ jobs:
         helixQueueGroup: ${{ parameters.helixQueueGroup }}
         ${{ insert }}: ${{ parameters.jobParameters }}
 
+# macOS arm64
+
+- ${{ if or(containsValue(parameters.platforms, 'OSX_arm64'), eq(parameters.platformGroup, 'all')) }}:
+  - template: xplat-setup.yml
+    parameters:
+      jobTemplate: ${{ parameters.jobTemplate }}
+      helixQueuesTemplate: ${{ parameters.helixQueuesTemplate }}
+      variables: ${{ parameters.variables }}
+      osGroup: OSX
+      archType: arm64
+      targetRid: osx-arm64
+      platform: OSX_arm64
+      jobParameters:
+        runtimeFlavor: ${{ parameters.runtimeFlavor }}
+        stagedBuild: ${{ parameters.stagedBuild }}
+        buildConfig: ${{ parameters.buildConfig }}
+        ${{ if eq(parameters.passPlatforms, true) }}:
+          platforms: ${{ parameters.platforms }}
+        # osx-arm64 is a cross build, but it doesn't require a ROOT_FS
+        # Use an identifiable value instead of creating and passing a separate parameter
+        crossrootfsDir: ':CrossBuildNoRootFS:'
```

Comment:
I waffled about this expedient.  An audit of existing crossrootfsDir allowed me confidence in this for the current instant...

I am happy to rework if there is objection to this hack...

Reply:
Can we just not pass the crossrootfsDir parameter instead?

Reply:
Yeah, let's not pass the parameter instead and it will be empty. Then if we need to set properties like crossBuild or what not we can do it based on osGroup and archType. We already cross build for iOS arm64 and we don't need to do this.

Reply:
I'll move the logic